controllers/temprole_controller.py

```diff
@@ -28,8 +28,15 @@ async def set_role(user: User | int, role: Role, duration: str) -> tuple[bool, s
             user_id = user.id
         else:
             user_id = user
-        delta = timedelta(seconds=timeparse(duration))
-        expiration = datetime.now() + delta
+        
+        try:
+            delta = timedelta(seconds=timeparse(duration))
+            expiration = datetime.now() + delta
+        except:
+            return (
+                False,
+                "Unable to assing role - please provide a time indicator for duration (e.g. s for seconds, m for minutes)"
```

Comment:
probably don't want to assing roles

Reply:
I'm not quite sure what you mean. Open to change things around if need be.